data/root/hostip_from_wicked

```diff
@@ -1,5 +1,13 @@
 #! /usr/bin/perl
 
+# helper script that
+#
+# (1) writes a list of assigned IP adresses to STDOUT, and
+#
+# (2) writes the FQDN or, if not available, the first IP to STDERR
```

Comment:
as this is about implementing new feature and not about a hotfix I'd go for more clean implementation.